app/models/admin_ability.rb

```diff
@@ -120,6 +125,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Registration, conference_id: conf_ids
     can :manage, RegistrationPeriod, conference_id: conf_ids
     can :manage, Booth, conference_id: conf_ids
+    cannot :accept, Booth do |booth|
```

Comment:
I see my mistake. I was testing it with a non-admin user and when the block was written in `signed_in_with_organizer_role`, the   `can :manage, Booth, conference_id: conf_ids_for_cfp` seemed to overwrite it (even if the block I wrote doesn't take into account of the user's role).

I defined the block in the common_abilities section because, although there is no `can :manage, Booth`, there is `can :manage, :all if user.is_admin`. (https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L54)
Only by putting the block there I could prevent an admin of accepting a booth when `booth_limit ` is reached. 

Taking that into consideration, I will leave the definition of the block in `common_abilities_for_roles` as it seems to be the only section that prevents admins, organizers, _and_ cfps from accepting a booth when they shouldn't.